### DISCUSSION TITLE
columnflow on lxplus

### DISCUSSION THREAD

**Discussion by JulesVandenbroeck** [columnflow/columnflow#551] [Q&A] [answered]
Dear columnflow developers,

I tried to setup the template columnflow analysis on lxplus.cern.ch but I ran into the following error in the setup:
```
setting up conda / micromamba environment
critical libmamba filesystem error: cannot set permissions: Permission denied [/afs/cern.ch/user/j/juvanden/columnflowProjects/test1/data/software/conda/pkgs/cache]
setup failed with code 1
```
I believe the setup of the template analysis should work without any changes on lxplus?
Here is the recipe to reproduce (which is quite straightforward):
 
 ```
 > bash -c "$(curl -Ls https://raw.githubusercontent.com/columnflow/columnflow/master/create_analysis.sh)"
 > Fill in the analysis name
 > cd 'analysis name'
 > source setup.sh
 ```
 
 Am I missing an important component which would resolve this issue?
 
 Kind Regards,
 Jules Vandenbroeck

**Comment by pkausw** [comment, marked as answer]
Hi @JulesVandenbroeck ,

indeed, the most straight-forward setup of columnflow on lxplus doesn't work at the moment. For whatever reason, micromamba can be installed on `/afs/` partitions, but is not allowed to install anything.

As mentioned in issue #552 , the solution is to set the `CF_SOFTWARE_BASE` variable to a directory in the /eos/ space, which seems to have no such issues. Can you try this in your setup as well?

Cheers,
Philip

**Reply by JulesVandenbroeck** [reply]
Thank you for the quick reply, this indeed solved the issue!
